Manage.c

```diff
@@ -1157,26 +1156,27 @@ int Manage_remove(struct supertype *tst, int fd, struct mddev_dev *dv,
 			/* Old kernels rejected this if no personality
 			 * is registered */
 			struct mdinfo *sra = sysfs_read(fd, NULL, GET_DEVS);
-			struct mdinfo *dv = NULL;
-			if (sra)
-				dv = sra->devs;
-			for ( ; dv ; dv=dv->next)
-				if (dv->disk.major == (int)major(rdev) &&
-				    dv->disk.minor == (int)minor(rdev))
-					break;
-			if (dv)
-				err = sysfs_set_str(sra, dv,
-						    "state", "remove");
-			else
+			struct mdinfo *dev = NULL;
+
+			if (!sra) {
 				err = -1;
-			sysfs_free(sra);
+			} else {
+				for (dev = sra->devs; dev ; dev=dev->next)
```

Comment:
Oh, last one:
```
ERROR: spaces required around that '=' (ctx:VxV)
#205: FILE: Manage.c:1164:
+                               for (dev = sra->devs; dev ; dev=dev->next)
                                                               ^

total: 1 errors, 0 warnings, 413 lines checked

```
review job is failed from different reason (workflows are configured to work from main branch and your head is outdated).